index.js

```diff
@@ -425,9 +424,9 @@ function fastifyMultipart (fastify, options, done) {
         return
       }
 
-      if (typeof opts.throwFileSizeLimit === 'boolean') {
-        throwFileSizeLimit = opts.throwFileSizeLimit
-      }
+      const throwFileSizeLimit = typeof options.throwFileSizeLimit === 'boolean'
+        ? options.throwFileSizeLimit
+        : defaultThrowFileSizeLimit
```

Comment:
From the comments in the file, `options` contains instance level configuration and `opts` contains per-request overrides. Hence the above conditional statement doesn't respect per-request override. `options` should be replaced with `opts`.

Reply:
Please provide a PR to discuss this ;)

Reply:
@Uzlopak opened #454.